package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "scripts": {
+    "start": "ts-node src/index.ts"
+  },
```

Comment:
How does this command work?

Reply:
You can simply type `yarn start` and your application will be executed.